src/js/components/Calendar/Calendar.js

```diff
@@ -32,13 +32,13 @@ import {
   daysApart,
   endOfMonth,
   formatToLocalYYYYMMDD,
-  localTimezoneToUTC,
   startOfMonth,
   subtractDays,
   subtractMonths,
   withinDates,
 } from './utils';
 import { CalendarPropTypes } from './propTypes';
+import { normalizeForTimezone, getTimestamp } from '../DateInput/utils';
```

Comment:
I tend to think of DateInput knowing about Calendar but Calendar being unaware of DateInput. To that end, I'm wondering if we should put common utils inside Calendar/utils.js

Reply:
I am comfortable with that. Was toggling between the two in my head